lib/spack/spack/environment/environment.py

```diff
@@ -773,9 +773,16 @@ def _read_manifest(self, f, raw_yaml=None):
         configuration = config_dict(self.yaml)
 
         # Let `concretization` overrule `concretize:unify` config for now.
-        unify = spack.config.get('concretizer:unify')
+        unify = spack.config.get('concretizer:unify', 'separately')
+        default_concretization = unify
+        if unify is True:
+            default_concretization = 'together'
+        elif unify is False:
+            default_concretization = 'separately'
```

Comment:
```suggestion
        try:
            translation = {'together': True, 'separately': False}
            self.concretization = translation[configuration['concretization']]
        except KeyError:
            self.concretization = spack.config.get('concretizer:unify', 'separately')

Reply:
I think this is a much cleaner way of letting the old way take precedence.

Reply:
Fixed in d7f254c